study.md

```diff
@@ -28,7 +28,7 @@ material. Instead, digest what you've read and repeat it in your own voice.
 Write the Ruby code that takes the integer `700` and returns the string `"007"`.
 
 ```ruby
-# your answer here
+"700".reverse
 ```
```

Comment:
the question makes this a bit confusing the way its worded but 700 starts as an integer so first we have to convert it to string, then reverse